neurom/check/morphtree.py

```diff
@@ -86,12 +87,26 @@ def is_flat(neurite, tol, method='tolerance'):
 
 
 def is_back_tracking(neurite):
-    ''' zigzag
+    ''' Check if a neurite process backtracks to a previous node. As back-tracking is taking place
```

Comment:
Shouldn't be there. I removed it. Also removed the detailed description of the algorithm.